Italian/guardare/libri-francesi/libri-francesi.md

```diff
@@ -6,3 +6,7 @@ I volumi sono sistemati con ordine maniacale, in rigoroso ordine alfabetico.
 Muovendo gli occhi lungo i diversi scaffali, incontri un libro stranamente non ricoperto di polvere, come se fosse
 stato letto da poco: "Les histoires horribles d'enfants chauves-souris", che suona qualcosa come "Le terribili storie
 dei bambini pipistrello".
+
+[Lasci perdere i libri e decidi di esaminare le fotografie, sperando di riconoscere qualcuno](../vecchie-fotografie/vecchie-fotografie.md)
+
+[Prendi il libro](/prendere-il-libro/prendere-il-libro.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.